scripts/utils/v4X_compatibility_test.sh

```diff
@@ -39,3 +40,9 @@ do
 done
 
 ./bootstrap; ./configure --enable-unit; make -j8 ; make check -j8
```

Comment:
For clarity it probably is better to split these out into their own lines

```suggestion
./bootstrap
./configure --enable-unit
make -j8
make check -j8
```